From: Zenghui Yu <zenghui.yu@linux.dev>
Subject: tools/mm: add thp_swap_allocator_test binary to .gitignore
Date: Wed, 24 Jun 2026 23:06:42 +0800

Tell git to ignore the generated binary for thp_swap_allocator_test.c.

Link: https://lore.kernel.org/20260624150642.19749-1-zenghui.yu@linux.dev
Signed-off-by: Zenghui Yu <zenghui.yu@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/mm/.gitignore |    1 +
 1 file changed, 1 insertion(+)

--- a/tools/mm/.gitignore~tools-mm-add-thp_swap_allocator_test-binary-to-gitignore
+++ a/tools/mm/.gitignore
@@ -2,3 +2,4 @@
 slabinfo
 page-types
 page_owner_sort
+thp_swap_allocator_test
_
